V2_MIGRATION_GUIDE.md

```diff
@@ -51,6 +51,86 @@ The `AuthenticationRequest` interface no longer has a `setAccessToken("{ACCESS-T
 
 Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequest` have been updated to accommodate these changes, and are called out in the detailed changes listed below.
 
+## Networking client customization
+
+v2 provides an improved ability to customize the way this library makes HTTP requests. A new interface, `NetworkingClient`, defines the contract for executing HTTP requests.
+The API clients and `WebAuthProvider` can be configured with an instance of `NetworkingClient`, making it possible to provide your own implementation for complete control over the networking client.  
+Additionally, the default networking client provided by this library supports several simple customization points to enable easy configuration of common customization points.
+
+### Timeout configuration
```

Comment:
I'm not 100% convinced of these as they look like a copy-paste of what you are already adding to the README in the "Networking client customization" section. In that case, it feels better a link to that README section rather than repeating yourself. 

That said, timeouts and logging were settings previously available, so I'd argue that they might fit this file. One thing to point out though is that these apply only when our `DefaultClient` implementation is being used. If someone implements their own, these will not apply to them. Is that something worth adding here?

Reply:
I kept the things that were possible before, but now changed. I did remove the section about advanced configuration, since that is not something that was supported previously.